server/src/main/kotlin/org/javacs/kt/resolve/ResolveMain.kt

```diff
@@ -15,7 +15,12 @@ fun resolveMain(file: CompiledFile): Map<String,Any> {
 
     findTopLevelMainFunction(parsedFile)?.let { mainFunction ->
         // the KtFiles name is weird. Full path. This causes the class to have full path in name as well. Correcting to top level only
-        parsedFile.name = parsedFile.name.partitionAroundLast("/").second.substring(1)
+        parsedFile.name = if (parsedFile.name.contains("/")) {
```

Comment:
Can we use `File.separator` (or some NIO Path equivalent) instead of manually checking for different kinds of slashes?